manifests/4.11/kubernetes-nmstate-operator.v4.11.0.clusterserviceversion.yaml

```diff
@@ -146,12 +146,12 @@ spec:
                   value: "6060"
                 - name: RUN_OPERATOR
                 - name: HANDLER_IMAGE
-                  value: quay.io/nmstate/kubernetes-nmstate-handler:latest
+                  value: quay.io/openshift/origin-kubernetes-nmstate-handler:4.11
```

Comment:
could the version  be not hardcoded?
We will need to update it manually for the 4.12

Reply:
This is the 4.11 manifest (see folder and file name). We have to add a separate 4.12 manifest.

Reply:
aaa,ok. I thought it could be some variable